cluster-provision/gocli/opts/istio/manifests/istio-operator.cr.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
   name: istio-operator
 spec:
   profile: demo
-  hub: quay.io/kubevirtci
   components:
```

Comment:
Please apply the same changes as in the `istio-operator-with-cnao.cr.yaml`, minus the `sidecarInjectorWebhook` part.